src/main/resources/com/puppycrawl/tools/checkstyle/checks/indentation/messages_zh.properties

```diff
@@ -1,5 +1,5 @@
 indentation.error.multi=''{0}'' 缩进了{1}个缩进符，应为以下缩进之一：{2}。
-indentation.child.error.multi=''{0}'' 子元素进了{1}个缩进符，应为以下缩进之一：{2}。
+indentation.child.error.multi=''{0}'' 子元素缩进了{1}个缩进符，应为以下缩进之一：{2}。
```

Comment:
It is a typo. I tried to use `endWith` way (I describe it at https://github.com/checkstyle/checkstyle/pull/3993#discussion_r105829337) in Chinese at first and found that typo. The `endWith` way will not work if the message in Chinese has a wrong postfix.